api/views/author.py

```diff
@@ -255,3 +255,19 @@ def get_serializer_context(self):
         context = super(GetFriends, self).get_serializer_context()
         context['nested_queryset'] = queryset
         return context
+
+
+class QueryAuthors(APIView):
+    """
+    Queries the database for all authors
+
+    """
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, *args, **kwargs):
+        authors =  Author.objects.all()
```

Comment:
Karim, I will probably just hook up an additional call that hits each node. Otherwise we only get some cached results which will only be there because of friending.